### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by localden**
Updates the XML comments with some documentation tweaks.

### CODE REVIEW

File: src/client/Microsoft.Identity.Client/PublicClientApplication.cs [hunk 1]
```diff
@@ -39,12 +37,13 @@ internal PublicClientApplication(ApplicationConfiguration configuration)
...
-        /// Currently only the Windows broker is able to login with the current user, see https://aka.ms/msal-net-wam for details.
-        /// 
-        /// A special account value that indicates that the current Operating System account should be used 
-        /// to login the user. Not all operating systems and authentication flows support this concept, in which 
+        /// A special account value that indicates that the current operating system account should be used 
+        /// to log the user in. Not all operating systems and authentication flows support this concept, in which 
         /// case calling `AcquireTokenSilent` will throw an <see cref="MsalUiRequiredException"/>. 
         /// </summary>
+        /// <remarks>
+        /// Currently, only the Windows broker is able to login with the current operating system user. For additional details, see <see href="https://aka.ms/msal-net-wam">documentation on the Windows broker</see>.
```
Comment (jmprieur) [inline src/client/Microsoft.Identity.Client/PublicClientApplication.cs:45]: Don't you want to use a <see> as well here?

File: src/client/Microsoft.Identity.Client/PublicClientApplication.cs [hunk 2]
```diff
@@ -117,29 +116,23 @@ public bool IsBrokerAvailable()
         }
 
         /// <summary>
-        /// Interactive request to acquire a token for the specified scopes. The interactive window will be parented to the specified
+        /// Interactive request to acquire a token for the specified scopes. The interactive window will be parented to the specified application
```
Comment (jmprieur) [inline src/client/Microsoft.Identity.Client/PublicClientApplication.cs:119]: application window?
do you mean the specified window of the app?

**Review by jmprieur — APPROVED** [review]
Awesome, @localden 
I left a couple of suggestions / questions

**Comment by trwalke** [comment]
Looks great overall, however, you should also propagate these changes to th `IPublicClientApplication` interface if you have not done so already.

File: src/client/Microsoft.Identity.Client/PublicClientApplication.cs [hunk 3]
```diff
@@ -74,29 +73,29 @@ internal static bool IsOperatingSystemAccount(IAccount account)
...
 
         /// <summary>
-        /// Returns true if MSAL can use an embedded web view (browser). 
+        /// Returns <c>true</c> if MSAL can use an embedded web view (web browser).
         /// </summary>
         /// <remarks>
-        /// Currently there are no embedded web views on Mac and Linux. On Windows, app developers or users should install 
-        /// the WebView2 runtime and this property will inform if the runtime is available, see https://aka.ms/msal-net-webview2
+        /// Currently, there are no embedded web views on macOS and Linux. On Windows, the WebView2 runtime is required and this property will reflect the availability of the component.
+        /// Refer to <see href="https://aka.ms/msal-net-webview2">our documentation</see> for additional details.
```
Comment (pmaytak) [inline src/client/Microsoft.Identity.Client/PublicClientApplication.cs:80]: This comment and WebView2 wiki aren't up-to-date.

On Windows
- NetFx comes with legacy web view
- Net6 and net core, don't have embedded web view, must use Msal.Desktop + WebView2.
- Net6-windows comes with WebView2 by default.

We also only show WebView2 UI for non-AAD authorities, AAD authorities get legacy web view.

**Review by pmaytak — APPROVED** [review]

**Review by trwalke — APPROVED** [review]

**Pull request merged**